.github/codeql/codeql-config.yml

```diff
@@ -1,10 +1,12 @@
 name: "Custom CodeQL Config"
 
 queries:
-- uses: security-and-quality
+  - uses: security-and-quality
 
 paths:
-- '**/extension/src/**'
+  - '**/extension/src/**'
 
 paths-ignore:
-- '**/.vscode-test/**'
+  - '**/.vscode-test/**'
+  - '**/AutoLispExt/out/**/*.js'
```

Comment:
this is already in another PR

Reply:
Yes, I need to merge that PR otherwise it will take very long time to run the codeql.